config_defaults.yml

```diff
@@ -159,6 +159,10 @@ Rakefile:
       Description: Prefer braces for chaining. Mostly an aesthetical choice. Better to be consistent then.
       EnforcedStyle: braces_for_chaining
 
+    Style/BracesAroundHashParameters:
+      Description: Braces are required by Ruby 2.7. Cop removed from RuboCop v0.80.0: https://github.com/rubocop-hq/rubocop/pull/7643
+      Enabled: false
```

Comment:
This is repetitive, but it seems good to have a "Description" that tells people why this one is disabled. Open to arguments against it though.